### DISCUSSION TITLE
UMEP URock - Study area size limitation

### DISCUSSION THREAD

**Discussion by pdugue** [UMEP-dev/UMEP#538] [Q&A]
Hi,

I am using URock for a large study area. I am satisfied by the results. However for future potential studies, I was thinking about size limitations for study areas. Is there any interpretation bias for very small or large study areas ? Is there a minimum or a maximum area size that have been identified, to remain more realistic as possible ?

Thanks, best,

Pauline

**Comment by j3r3m1** [comment]
We have not investigated in detail this question yet but even though URock is not a prognostic model such as most of the CFD models, I suggest you to use as basic recommendations the ones proposed by [Tominaga et al. (2008)](https://www.sciencedirect.com/science/article/abs/pii/S0167610508000445?via%3Dihub) (section 3.2) which are dedicated to pedestrian wind comfort studies:

_For the actual urban area, the buildings in the region to be assessed (generally 1–2H radius from the target building) should be clearly modeled. Moreover, at least one additional street block in each direction around the assessment region should also be clearly reproduced (Yoshie et al., 2005a, b). In addition, it is recommended to use some simplified geometries of a cluster of buildings or to specify appropriate roughness lengths zo for the ground surface boundary condition to represent the roughness of the outer region (from the outer edge of the additional street blocks to the boundary of the computational domain)_
